Lesson16_Task01.py

```diff
@@ -0,0 +1,136 @@
+"""
+Home-task 13 Class writing Practice
+
+# write class for bank card.
+# Class should reflect card lifecycle, card operations etc.
+# use CVV, PIN, Name, Surname , end date, card_num as initial params
+# class should have in addition to common logic some class attributes, as minimum one classmethod and
+# as minimum  one staticmethod, two or more getters/setters
+# do not forget about block '''if __name__ == '__main__':'''
+"""
+
+import datetime
+
+
+class BankCard:
```

Comment:
Your class according task requirements should have at least 1 classmethod - please add it

Reply:
Still can't find classmethod example in your code